pkg/observability/utils/growl.ts

```diff
@@ -0,0 +1,13 @@
+
+export interface HandleGrowlConfig {
+  type: string;
+  message: string;
+}
+
+export function handleGrowl(store: any, { type, message }: HandleGrowlConfig) {
```

Comment:
Careful that a growl might not always be an error. In stackstate's case, it looks like you are just using it for error display, which makes this fine, but it can have other applications.

Reply:
true, so far there are only error messages and it can extend in future as soon as another case appers